src/handlers/InvitesBtn.ts

```
@@ -1,21 +1,25 @@
 import { database } from "../utils/base"
 import { createGenericEmbed } from "../utils/embeds"
 import { InvitesActionId, InvitesId } from "../utils/interactionIds"
-import { checkChannelRights } from "../utils/preconditions"
+import { ChannelNotFound, checkChannelRights, CheckRights } from "../utils/preconditions"
+import { rerenderDashboard } from "../utils/misc"
+import { inviteUser } from "../utils/inviteStatus"
 
 @ButtonRoute(InvitesId)
 export class InvitesButton extends ButtonHandler<[typeof InvitesId]> {
     public async execute(): Promise<void> {
         if (this.event.channel?.type != ChannelType.GuildVoice) return
```

ESLint

[error] Expected '!==' and instead saw '!='.

(eqeqeq)